Dockerfile

```diff
@@ -15,6 +15,8 @@ COPY . /app
 
 
 FROM $base_image
+RUN apt-get update -qq && apt-get upgrade -y
+RUN apt-get install -y libpq-dev && apt-get clean
```

Comment:
@KludgeKML Do you think you might need a puppet entry for postgres in authenticating proxy?

Reply:
e.g. Similar to: https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk/manifests/apps/content_publisher/db.pp and https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk/manifests/apps/content_publisher.pp#L215-L222 to set the env vars?

Reply:
Ah, yes - I think in between me writing this and me rebasing it a workflow got added, and of course it's Mongo. (and double of course, the govuk reusable actions don't support postgres). Let me see if I can roll my own here.

Reply:
Ugh. Looks like there's a whole bunch of issues. For the moment none of the apps that use PG have github actions, so I'm going to assume that at some point someone will add it to the reusable action. For now, jenkins only.

Reply:
This is fixed, github actions working now.